### PR TITLE
Refactor install

### PR DESCRIPTION
**Pull request by jpa198**

### CODE REVIEW

File: shared_workflow/install_shared.py [hunk 1]
```diff
@@ -215,7 +270,57 @@ def install_simulation(
...
+    root_params_base_dict, root_folder, extended_period, seed, stat_file_path, version
+):
+    root_params_base_dict["mgmt_db_location"] = root_folder
+    root_params_base_dict[RootParams.extended_period.value] = extended_period
+    root_params_base_dict[RootParams.version.value] = version
+    root_params_base_dict[RootParams.stat_file.value] = stat_file_path
+    root_params_base_dict[RootParams.stat_vs_est.value] = stat_file_path.replace(
+        ".ll", ".vs30"
+    )
+    root_params_base_dict[RootParams.stat_vs_ref.value] = stat_file_path.replace(
```
Comment (jasonmotha) [inline shared_workflow/install_shared.py]: @shot0829 Do we need this one?

File: scripts/cybershake/install_cybershake.py [hunk 2]
```diff
@@ -108,16 +115,17 @@ def main():
...
-            fault,
-            count,
-            path_cybershake,
-            args.version,
-            args.stat_file_path,
-            args.seed,
-            args.extended_period,
-            workflow_logger.get_realisation_logger(logger, fault),
-        )
+        for i in range(1, count + 1):
```
Comment (jasonmotha) [inline scripts/cybershake/install_cybershake.py]: Comment to explain magic number

File: shared_workflow/install_shared.py (hunk 1 above)
Comment (shot0829) [reply shared_workflow/install_shared.py]: Its not being used anymore. And there is a way to emulate the same feature by specifying a value of lfvsref under bb. So i would say its good to be removed.

**Comment by jpa198** [comment]
Yeah, the refactor is in no mans land where installation is set up to be in the workflow, but isn't yet, so nothing will work

**Pull request merged**